liquibase-maven-plugin/src/test/java/org/liquibase/maven/plugins/LiquibaseTagExistsMojoTest.java

```diff
@@ -0,0 +1,59 @@
+package org.liquibase.maven.plugins;
+
+import org.apache.maven.plugin.MojoFailureException;
+import org.codehaus.plexus.configuration.PlexusConfiguration;
+
+import java.util.HashMap;
+import java.util.Map;
+
+/**
+ * Some basic tests that validate the setting of properties for the
+ * LiquibaseTagExistsMojo.
+ */
+public class LiquibaseTagExistsMojoTest extends AbstractLiquibaseMojoTest {
+
+	// reusing dropAll/plugin_config.xml since the LiguibaseTagExists mojo does
```

Comment:
I appreciate the comment explaining why this particular changelog is used in the test. Thank you!